internal/otc_wrapper.go

```diff
@@ -289,13 +299,17 @@ func (c *OtcWrapper) GetMetricDataBatched(metrics []otcMetrics.MetricInfoList) (
 	return result, err
 }
 
-func FilterByNamespaces(metrics []otcMetrics.MetricInfoList, namespaces []string) []otcMetrics.MetricInfoList {
+func (c *OtcWrapper) FilterByNamespaces(metrics []otcMetrics.MetricInfoList, namespaces []string) []otcMetrics.MetricInfoList {
```

Comment:
I would like this function to stay a function and not become a method. The reason for this is that the reponsibility of the OtcWrapper is to make it easier to call the otc api. This function performs a filter on a slice.

The question here is about logging. On what granularity do we need logging?
Is it maybe better to log the resulting set? Thereby I mean the metrics which are included?

Reply:
So I mostly included logging here to explicitly tell me which metrics were
being excluded, since I was searching for the reason DDS metrics weren't
available.

This logging here could either be removed entirely, or replaced with
logging of the sets before and after filtering. Or we could modify the
function to return both the filtered slices, and the slices that were
filtered out, which would give the same level of granularity in the debug
logging if needed?